benchmarks/asv/optimize.py

```diff
@@ -44,30 +47,49 @@ def objective(self, trial: optuna.Trial) -> float:
         y = trial.suggest_int("y", -100, 100)
         return x**2 + y**2
 
-    def optimize(self, storage_mode: str, sampler_mode: str, n_trials: int) -> None:
+    def multi_objective(self, trial: optuna.Trial) -> tuple[float]:
+        x = trial.suggest_float("x", -100, 100)
+        y = trial.suggest_int("y", -100, 100)
+        return (x**2 + y**2, (x - 2) ** 2 + (y - 2) ** 2)
+
+    def optimize(
+        self, storage_mode: str, sampler_mode: str, n_trials: int, objective_type: str
+    ) -> None:
         with StorageSupplier(storage_mode) as storage:
             sampler = create_sampler(sampler_mode)
             study = optuna.create_study(storage=storage, sampler=sampler)
```

Comment:
Could you specify `directions` for multi-objective? Currently, all trials are failed.

Reply:
You can specify the `directions` as follows.
```suggestion
            if objective_type == "single":
                directions = ["minimize"]
            elif objective_type == "multi":
                directions = ["minimize", "minimize"]
            else:
                assert False
            study = optuna.create_study(storage=storage, sampler=sampler, directions=directions)
```

Reply:
Thank you for your advice.